Marlin/Marlin_main.cpp

```diff
@@ -4130,6 +4143,39 @@ inline void gcode_M226() {
   }
 #endif
  
+#ifdef DAC_I2C
+  /**
+   * M236 - Send Value to ADC w/ no EEPROM write *TESTING*
+   */
+  inline void gcode_M236() {
+    if(code_seen('S')) {
```

Comment:
Agreed, but as is it will work properly. I will add this functionality very soon.